nix/cells/automation/pipelines.nix

```diff
@@ -27,13 +25,15 @@ in
   ci = { config, lib, ... }: {
     imports = [ common ];
 
-    preset.github-ci.sha = config.preset.github-ci.lib.readRevision
+    preset.github.ci.revision = config.preset.github.lib.readRevision
       inputs.cells.cloud.library.actions.ci.input
       null;
 
-    command.text = ''
-      nix build -L .#${lib.escapeShellArg system}.automation.ciJobs.required
-    '';
+    command.text = config.preset.github.status.lib.reportBulk {
+      bulk.text = "nix eval .#__std.init --apply __attrNames --json | nix-systems -i";
+      each.text = ''nix build -L .#"$1".automation.ciJobs.required'';
```

Comment:
What is `$1`? What does any of this do? Please write some comments!

Reply:
I did look at the definition of `reportBulk`, but it's undocumented so that didn't help. You could document that instead!

Reply:
Does this help? https://github.com/input-output-hk/tullia/commit/0a98e231cdec070d3771eb27c73e1577a5b5895a